spec/factories/books.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :book do
+    title "MyString"
```

Comment:
@valentina91p Done. In my last pr you should find all the changes you asked for.